doc/user/content/transform-data/_index.md

```diff
@@ -9,18 +9,13 @@ menu:
     identifier: transform-overview
 ---
 
-With Materialize, you can use SQL to transform, deliver, and act on
```

Comment:
I wouldn't replace the blurb in this page. You could just link to the page above from "compatibility with the PostgreSQL dialect", rather than repeating the whole thing.